src/ui/Badge/Badge.tsx

```diff
@@ -2,6 +2,8 @@ import cs from 'classnames'
 
 const BadgeVariants = {
   default: 'bg-ds-pink-default text-white',
+  danger: 'bg-white text-ds-primary-red border border-ds-primary-red',
```

Comment:
we may need to update these bg colors for dark mode equivalents

Could potentially create a new class in globals.css called "card-background" or similar

Reply:
so just the `bg-white`?

Reply:
i actually don't need it, it's by default white and it inherits the dark mode styles.

Reply:
its always great when we can delete code AND get stuff for free 😂